packages/cacti-weaver-driver-fabric/src/main/typescript

```diff
@@ -0,0 +1 @@
+../../../../weaver/core/drivers/fabric-driver
```

Comment:
@sandeepnRES Was this meant to be a directory instead of a file by any chance?

Reply:
It is a symbolic link file to weaver directory

Reply:
@sandeepnRES Oh, OK, got it! Sorry for not coming back to this earlier, please hit the re-request review button next time otherwise it doesn't show up in my review queue!